packages/admin/admin/src/form/FinalFormToggleButtonGroup.tsx

```diff
@@ -37,8 +49,10 @@ const Root = styled("div", { shouldForwardProp: (prop) => prop !== "$optionsPerR
 `;
 
 const Button = styled(ButtonBase, { shouldForwardProp: (prop) => prop !== "$selected" })<{ $selected?: boolean }>`
-    width: 46px;
-    height: 46px;
+    padding-left: ${({ theme }) => theme.spacing(3)};
+    padding-right: ${({ theme }) => theme.spacing(3)};
+    padding-top: ${({ theme }) => theme.spacing(1.8)};
+    padding-bottom: ${({ theme }) => theme.spacing(1.8)};
```

Comment:
I'd just not use the spacing-helper if the design doesn't use them 😄 

```suggestion
    padding-top: 9px;
    padding-bottom: 9px;
```

Unless the idea here that it's exactly `2` spacings, but excluding `1px` for the border, the that should explicitly be set: 

```suggestion
    padding-top: ${({ theme }) => `calc(${theme.spacing(2)} - 1px)`};
    padding-bottom: ${({ theme }) => `calc(${theme.spacing(2)} - 1px)`};
```